west.yml

```diff
@@ -57,7 +57,7 @@ manifest:
       groups:
         - hal
     - name: hal_espressif
-      revision: 73e7af1e2ed64571ce49ff9f07dc02690b9f2df5
+      revision: e41b0876c8bb0b9f90846c14397beff52d5590b4
```

Comment:
this is pulling in a lot more than a pin fix. Is this intended?

Reply:
@carlocaione, indeed, it also brings latest ESP32-S3 content, which is also at #53699. I think that one needs to be merged 1st? Well, this can be merged as is. Perhaps I can update the commit message only?

Reply:
I would wait for the #53699 to be merged before this one.

Reply:
> I would wait for the #53699 to be merged before this one.

Sounds good, thanks.